pkg/kubeclient/signup.go

```diff
@@ -77,27 +83,42 @@ func (c *userSignupClient) Update(obj *crtapi.UserSignup) (*crtapi.UserSignup, e
 	return result, nil
 }
 
-func (c *userSignupClient) ListByPhone(phone string) (*crtapi.UserSignupList, error) {
-	return c.listByHashedLabel(crtapi.BannedUserPhoneNumberHashLabelKey, phone)
-}
+// ListByPhoneNumberOrHash will return a list of UserSignups that have a phone number hash label value matching
+// the provided value.  If the value provided is an actual phone number, then the hash will be calculated and then
+// used to query the UserSignups, otherwise if the hash value has been provided, then that value will be used
+// directly for the query.
+func (c *userSignupClient) ListByPhoneNumberOrHash(phoneNumberOrHash string) (*crtapi.UserSignupList, error) {
```

Comment:
And here (not deactivated only).
And we will need to mention it in the comment.

Reply:
Renamed to `ListActiveSignupsByPhoneNumberOrHash()`